README.md

```diff
@@ -1 +1,158 @@
-# figma_layout_grid
+# Figma layout grid
+```figma_layout_grid``` is a Flutter package which imitates Figma's layout grid and all its functionalities. It helps you determine whether your UI components match the design file and align properly. 
+
+## Basic usage
+
+Simply wrap a widget you wish to have layout grid rendered on top with ```LayoutGrid```. Except that you have to pass three required parameters that will indicate what layers have to be shown.
+
+
+```dart
+LayoutGrid(
+    showColumns: GridConfig.showColumnsNotifier,
+    showRows: GridConfig.showRowsNotifier,
+    showGrid: GridConfig.showGridNotifier,
+    child: const MyHomePage(),
+)
+```
+
+Note that you can export ```showColumns```, ```showRows``` and ```showGrid``` parameters to a separate config class just like that.
+
+```dart
+class GridConfig {
+  static final showColumnsNotifier = ValueNotifier(false);
+  static final showRowsNotifier = ValueNotifier(false);
+  static final showGridNotifier = ValueNotifier(false);
+}
+```
+
+## Available layouts
+```figma_layout_grid``` supports three types of layout overlays - rows, columns and pixel grid.
+
+### Rows
+
+Horizontal stripes with specified height, gutter, margin, offset, arrangement and color.
+There is a possibility for fixed rows count with passing desired number in a ```count``` parameter, otherwise rows are rendered dynamically due to the screen limits. 
+
+Example usage
+
+```dart
+LayoutGrid(
+  ...
+  showRows: GridConfig.showRowsNotifier,
+  rowsParams: const RowsParams(
+    offset: 32.0,
+    height: 16.0,
+    gutter: 16.0,
+  ),
+  child: const MyHomePage(),
+),
+```
+
+<img src="resources/rows.png" alt="drawing" style="width:300px;"/>
```

Comment:
it will not work on pub.dev you need to full link to image on github. Also add better alt